website/docs/docs/deploy/understanding-state.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Understanding state"
+title: "Understand state"
```

Comment:
```suggestion
title: "Understand project state"
```

Reply:
sugg:
- consider "About state" for consistency with other pages since it's a concept 
- consider "Understand project state" (or "About project state" with ^) to be more _descriptive_. if you wanna incorporate this sugg, make the other changes too like redirect, page routing, sidebar label, etc for consistency

Reply:
changing to 'about state'